### ISSUE TITLE
Parallel + split working with server, but not regular test

### ISSUE THREAD

**Issue by samselikoff** [trentmwillis/ember-exam#122]
I just installed ember-exam on a new project. Running

```
ember exam -s --split=4 --parallel
```

works great (4 Chrome instances open up), but running

```
ember exam --split=4 --parallel
```

doesn't seem to parallelize anything. Only one browser opens and my tests just run through to completion.

Any ideas on how to diagnose?

**Comment by lolmaus** [comment]
Did you add `loadEmberExam()` to `test-helper.js`?

See https://github.com/trentmwillis/ember-exam#how-to-use .

**Comment by trentmwillis** [comment]
Sounds like it is potentially a Testem bug, as the processing that Ember Exam does is the same for both modes. Can't say much more than that unfortunately. A reproduction would be helpful if the issue is still occurring.

**Comment by samselikoff** [comment]
@lolmaus yep, and parallelization does work with `ember exam -s`. Just not `ember exam`.

**Comment by samselikoff** [comment]
The issue was with the `parallel` flag not being set in `testem` config. There's a note about this but I missed it. Can I PR the docs to move the note to the top of the "Split" section?

**Comment by lolmaus** [comment]
Same [happened](https://github.com/trentmwillis/ember-exam/issues/108) to me. The installation instructions do need to be grouped in one Readme section and emphasized more.

@samselikoff Looking forward to your edit.

**Comment by BobrImperator** [comment]
For anyone encounterint this problem, this resolved the problem for me.
https://github.com/ember-cli/ember-exam/issues/270
